crates/sui-types/src/gas_model/tables.rs

```diff
@@ -340,9 +340,9 @@ impl GasStatus {
     }
 
     pub fn log_for_replay(&self) {
```

Comment:
I understand this is only called for replay and we have some kind of precedent here. I also understand it may not be easy to access this info from completely the outside, but it would be nice to have this available as data that we can decide to format different according to context.
Maybe not possible but let's spend some cycle thinking about it